main.rb

```diff
@@ -1,10 +1,49 @@
-require_relative 'person'
-require_relative 'trimmer_decorator'
-require_relative 'capitalize_decorator'
+#!/usr/bin/env ruby
+require './app'
+def options
+  puts ''
+  puts 'Please choose an option by entering a number'
+  puts '1 - List all books.'
+  puts '2 - List all people'
+  puts '3 - Create a person.'
+  puts '4 - Create a book.'
+  puts '5 - Create a rental.'
+  puts '6 - Create List all rentals for a given person id.'
```

Comment:
[OPTIONAL] I made this optional because your app works perfectly and all functions and options work just as expected. I believe option 6 should be ```"List all rentals for a given person id"``` instead of ```"Create list all rentals for a given person id"```. It may be a minor change but little grammatical errors can really affect the UX of your page and I encourage you to prevent them as much as possible 😉

Reply:
@amon-cofie, thank you for your feedback. I have corrected the mistake.